### ISSUE TITLE
Problem with get

### ISSUE THREAD

**Issue by jmarroyave** [joshjung/hash-array#3]
Hi Josh i am using the trie-search and found a bug in it when using reserved words as a key, after debugging i found out that the problem was in the function 

this.getCache.get(phrase)

when phrase = 'constructor' it returns a function() instead of an element or array, hope it clear enough to let you find out which the problem is.

Thanks in advance

**Comment by tunnckoCore** [comment]
Yup. Easiest fix is updating L16 to `this._map = Object.create(null);` this way later when you are accessing `this._map[key]` (in lot of places) you won't have problems such as with `constructor`.

/cc @joshjung 

Great library!

**Comment by joshjung** [comment]
@tunnckoCore or @jmarroyave I'm rather busy with other projects at the moment. If you make a PR for this I'll make sure to do a new release!

**Comment by tunnckoCore** [comment]
@joshjung cool, thanks. I can do it in coming days.

**Comment by 7iomka** [comment]
;)
